src/java.rmi/share/classes/sun/rmi/runtime/NewThreadAction.java

```diff
@@ -25,13 +25,11 @@
 
 package sun.rmi.runtime;
 
-import java.security.AccessController;
-import java.security.PrivilegedAction;
-import sun.security.util.SecurityConstants;
```

Comment:
Are there any remaining uses of sun.security.util.* in java.rmi? I suspect this is the last one, in which case you can update java.base's module-info to not export this package to java.rmi.

Reply:
I will check if it is the last one as part of [JDK-8344299](https://bugs.openjdk.org/browse/JDK-8344299).

Reply:
Roger has filed [JDK-8344867](https://bugs.openjdk.org/browse/JDK-8344867) to cover cleanup of the unneeded qualified export.